pkgs/by-name/yu/yunfaavatar/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  bash,
+  imagemagick,
+}:
+stdenvNoCC.mkDerivation rec {
+  pname = "yunfaavatar";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "yunfachi";
+    repo = "yunfaAvatar";
+    rev = version;
+    hash = "sha256-s381fD+Zr4CdeHCz8glvC+0wnDiFzn32/iFSrc5p05U=";
+  };
+
+  strictDeps = true;
+  buildInputs = [bash imagemagick];
```

Comment:
These are used at runtime, correct? If so, you need to use `makeWrapper` to add them to PATH for the actual executable.

Reply:
`imagemagick` here, https://github.com/yunfachi/yunfaAvatar/blob/be8d383b6eba41a37e52f0e3128ed3a234d4875f/utils/avatar.sh#L31

(also the whole things a bash script)